doc/api/esm.md

```diff
@@ -703,13 +703,13 @@ a URL should be interpreted, retrieved, and parsed.
 
 The final value of `format` must be one of the following:
 
-| `format`     | Description                    | Acceptable types For `source` returned by `resolve` or `load` |
-| ------------ | ------------------------------ | -------------------------------------------------------------------------- |
-| `'builtin'`  | Load a Node.js builtin module  | Not applicable                                                             |
-| `'commonjs'` | Load a Node.js CommonJS module | Not applicable                                                             |
-| `'json'`     | Load a JSON file               | { [`string`][], [`ArrayBuffer`][], [`TypedArray`][] }                      |
-| `'module'`   | Load an ES module              | { [`string`][], [`ArrayBuffer`][], [`TypedArray`][] }                      |
-| `'wasm'`     | Load a WebAssembly module      | { [`ArrayBuffer`][], [`TypedArray`][] }                                    |
+| `format`     | Description                    | Acceptable types for `source` returned by `load`      |
```

Comment:
It seems involving multlple unrelated changes. They should be removed. 

We could do a rebase then force push changes in the PR.

Reply:
> It seems involving multlple unrelated changes. They should be removed.

Thank you for your review. I have removed it.